sdk/src/main/scala/com/horizen/certificatesubmitter/keys/SchnorrKeysSignaturesListBytes.scala

```diff
@@ -1,8 +1,6 @@
 package com.horizen.certificatesubmitter.keys
 
-import com.horizen.schnorrnative.{SchnorrPublicKey, SchnorrSignature, ValidatorKeysUpdatesList}
-
-import java.util
+import com.horizen.utils.BytesUtils
 
 case class SchnorrKeysSignaturesListBytes(
```

Comment:
It was a comment before abouth this class
1) must be renamed, arguments as well
2) forget about bytes -> use SchnorrProposition class

Find the previous commend and address the changes
@sgonta

Reply:
Done